source/CMakeLists.txt

```diff
@@ -153,6 +163,9 @@ if(NOT SKBUILD)
         WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
 endif()
 
+add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/libtomo/misc)
+add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/libtomo/prep)
+
```

Comment:
Please move these up to before the python files are copied to binary directory.

Reply:
The common method is to do this:

```console
add_subdirectory(libtomo)
```

and then add another CMakeLists.txt in that folder that does:

```console
add_subdirectory(misc)
add_subdirectory(prep)
```